planetiler-core/src/main/java/com/onthegomap/planetiler/mbtiles/MbtilesWriter.java

```diff
@@ -59,6 +65,18 @@ public class MbtilesWriter {
   private final FeatureGroup features;
   private final AtomicReference<TileCoord> lastTileWritten = new AtomicReference<>();
   private final MbtilesMetadata mbtilesMetadata;
+  private final AtomicLong tileDataIdGenerator = new AtomicLong(1);
+  //  private final LoadingCache<ByteBuffer, Long> tileDataIdByHash = CacheBuilder.newBuilder()
```

Comment:
de-duping over a shared cache would allow to reduce the size of Australia by another ~100MB but it adds some overhead and more importantly it would also make the output again less predictable
=> would like to get your input here